apps/web/public/static/locales/en/common.json

```diff
@@ -2160,5 +2160,6 @@
   "need_help": "Need help?",
   "troubleshooter": "Troubleshooter",
   "please_install_a_calendar": "Please install a calendar",
+  "meeting_started": "Meeting Started",
```

Comment:
Suggestion : "meeting_started": "Meeting started", - "started" to start with lower case "s"

Reply:
It should be uppercase like all the other webhooks as well: 
![Screenshot 2023-12-19 at 8 47 24 AM](https://github.com/calcom/cal.com/assets/30310907/4e5501d4-87a9-4c6b-9825-406134cfe1fe)